### PR TITLE
App bootstrapped

### PR DESCRIPTION
**Pull request by tedscape**
HI Shidhin,
The app has been bootstrapped and tested. Can you check and approve the pull request.?
Thanks & Best Regards,
Jacob Sebastian

### CODE REVIEW

File: .idea/MaterialCalc.iml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shidhincr) [inline .idea/MaterialCalc.iml:1]: We should exclude all the editor files to .gitignore

add `.idea`  to **.gitgnore** file

**Comment by shidhincr** [comment]
@tedscape How did you bootstrap ? Did you use the react-native tool ?

**Comment by tedscape** [comment]
@shidhincr I used the steps in https://facebook.github.io/react-native/docs/getting-started.html#content

**Comment by shidhincr** [comment]
You also don't need to fork the project and open the PR from there.. I already made you as the collaborator, so you can clone this repo and create branches..

Then push your changes to the branch

**Comment by tedscape** [comment]
Okie. You can approve the PR now. Hope I have done it right. Messed up with github for Mac.
 I will clone your repo for next tasks.

**Pull request merged**